gplugins/SiEPIC/verification.py

```diff
@@ -0,0 +1,62 @@
+# SiEPIC - can be loaded from PyPI or in KLayout Application, or loaded from a local folder such as GitHub
+import os
+import sys
+
+path_GitHub = "/Users/lukasc/Documents/GitHub/"
```

Comment:
Hardcoded path?

Reply:
Hi Joaquin,
The hardcoded path section is temporary to use the module from GitHub, rather than the pip-installed one. It can be removed.